tests/test_config.py

```diff
@@ -0,0 +1,35 @@
+from unittest.mock import Mock, patch
```

Comment:
This causes an error
E   ImportError: cannot import name 'config' from partially initialized module 'app' (most likely due to a circular import) (/home/alan/devel/langchain-demo/cai-platform/app/__init__.py)